backend/backend/src/main/java/de/aaaaaaah/velcom/backend/access/TaskWriteAccess.java

```diff
@@ -168,10 +166,10 @@ void insertTasks(Collection<Task> originalTasks, DSLContext db) {
 	 * @param taskIds the ids of the tasks to delete
 	 */
 	public void deleteTasks(Collection<TaskId> taskIds) {
-		deleteTasks(taskIds, databaseStorage.acquireContext());
+		deleteTasks(taskIds, databaseStorage.acquireWriteAccess());
```

Comment:
Why is no try-with-resources used here?

Reply:
Fixed in https://github.com/IPDSnelting/velcom/commit/9c85694d224e55f310d9330460dca600ea140c58